cla_frontend/apps/status/views.py

```diff
@@ -27,11 +31,27 @@ def status(request):
     })
 
 
-def smoketests(request):
+def ping(request):
+    return JSONResponse({
+        'version_number': os.environ.get('APPVERSION'),
+        'build_date': os.environ.get('APP_BUILD_DATE'),
+        'commit_id': os.environ.get('APP_GIT_COMMIT'),
+        'build_tag': os.environ.get('APP_BUILD_TAG')
+    })
+
+
+def smoketests_json(request):
     """
     Run smoke tests and return results as JSON datastructure
     """
 
     from cla_frontend.apps.status.tests.smoketests import SmokeTests
 
     return JSONResponse(smoketest(SmokeTests))
+
+
+def healthcheck_json(request):
+
+    backend_healthcheck_uri = '%s/%s' % (settings.BACKEND_BASE_URI, 'status/healthcheck.json')
+    response = requests.get(backend_healthcheck_uri)
```

Comment:
Should check the response code and return its own healthcheck rather than the backend one. Also should just use healthcheck view from moj_irat. Healthchecks should be in a healthcheck.py